test/functional/test_helper.py

```diff
@@ -37,3 +39,21 @@ def is_ci() -> bool:
         `True` if current executions is on CI
     """
     return os.getenv('CI', 'false') == 'true'
+
+
+def wait_for(method: Callable, timeout: int) -> bool:
+    """Wait for `method` True
+
+    Args:
+        method: The target method to be waited. Need to return `bool`
+        timeout: The timeout to be waited (sec.)
+
+    Returns:
+        bool: True if `method` returns True
+
+    """
+    for i in range(timeout + 1):
+        if method():
+            return True
```

Comment:
I would rather return the result of the method itself. this should not necessarily be a boolean value

Reply:
@mykola-mokhnach 

Humm, in my understanding, I'm not sure how it can wait for target `Callable` if it isn't boolean.
Would you give more details?